src/main/java/com/undertheriver/sgsg/foler/controller/FolderController.java

```diff
@@ -25,49 +27,43 @@
 @RequestMapping("/api/v1/folders")
 @Api(value = "folder")
 public class FolderController {
-    private final FolderService folderService;
+	private final FolderService folderService;
 
-    @ApiOperation(value = "폴더 생성")
-    @PostMapping
-    public ApiResult<?> save(
-            @LoginUserId Long userId,
-            @RequestBody FolderDto.CreateFolderReq dto) {
-        try {
-            Long id = folderService.save(userId, dto);
-            URI location = new URI("/api/folders/" + id);
-            return ApiResult.OK(location);
-        } catch (IndexOutOfBoundsException e) {
-            return ApiResult.ERROR(e.getMessage(), HttpStatus.NOT_ACCEPTABLE);
-        } catch (Exception e) {
-            return ApiResult.ERROR(e.getMessage(), HttpStatus.BAD_REQUEST);
-        }
-    }
+	@ApiOperation(value = "폴더 생성")
+	@PostMapping
+	public ApiResult<?> save(
+		@LoginUserId Long userId,
+		@RequestBody FolderDto.CreateFolderReq dto) throws URISyntaxException {
+		long id = folderService.save(userId, dto);
+		URI location = new URI("/api/folders/" + id);
```

Comment:
내부에서 URI을 만들어주는데 checked exception이 날 필요는 없을 것 같아요.
```suggestion
		URI location = URI.create("/api/folders/" + id);
```
로 변경하신다면 URISyntaxException을 사용한 곳이 모두 사라질 수 있을 것 같네요